buildconfig/windows/pycharm.bat.in

```diff
@@ -3,6 +3,9 @@ setlocal enableextensions enabledelayedexpansion
 
 set KEY_NAME="HKCR\Applications\pycharm.exe\shell\open\command"
 
+reg query %KEY_NAME% 2> nul
+if %ERRORLEVEL% NEQ 0 set KEY_NAME="HKCR\Applications\pycharm64.exe\shell\open\command"
```

Comment:
I know this is fairly painful in .bat files as there is no `and` operand but could we do

```
SET PYCHARM_32_KEY_NAME="HKCR\Applications\pycharm.exe\shell\open\command"
SET PYCHARM_64_KEY_NAME="HKCR\Applications\pycharm64.exe\shell\open\command"


reg query %PYCHARM_32_KEY_NAME% 2> nul
IF %ERRORLEVEL% NEQ 0  (
    reg query %PYCHARM_64_KEY_NAME% 2> nul
    IF %ERRORLEVEL% NEQ 0 (
        rem Print an error here telling the user their 32 and 64 bit installations could not be found
        exit -1
    ) ELSE (
        SET KEY_NAME="HKCR\Applications\pycharm64.exe\shell\open\command"
    )
)
```
(There's a tiny nit pick that in .bat files the convention is to capitalize all batch keywords too)

Reply:
I've made the changes to the batch file. Had a bit of fun with the ERRORLEVEL not being re-evaluated but got there in the end.
Re the capitals\not capitals convention - I've gone with keywords not capitalized and variable names capitalized because this was the approach used in most other mantid .bat files I found eg launch_mantidplot.bat. No problem to change to caps though if you insist.

Reply:
Thanks for struggling through that. 
If the project has used a different convention, we should probably stick with it. I doubt many people will look into this file again